tracer/build/_build/Build.VariableGenerations.cs

```diff
@@ -656,7 +672,9 @@ void GenerateLinuxNuGetSmokeTestsMatrix()
                         "alpine",
                         new (string publishFramework, string runtimeTag)[]
                         {
-                            (publishFramework: TargetFramework.NET7_0, "7.0-alpine3.16"),
+                            (publishFramework: TargetFramework.NET8_0, "8.0-alpine3.18"),
+                            // (publishFramework: TargetFramework.NET8_0, "8.0-alpine3.18-chiseled"), // we can't run scripts in chiseled containers, so need to update the dockerfiles
+                            // (publishFramework: TargetFramework.NET7_0, "7.0-alpine3.16"), // we can't run scripts in chiseled containers, so need to update the dockerfiles
```

Comment:
Is 7.0 commented out on purpose?

Reply:
Yes, currently this doesn't work IIRC 😅 I need to remind myself exactly why, but either way I think we should address it in a subsequent PR 🙂